torch/testing/_internal/common_utils.py

```diff
@@ -1231,7 +1231,7 @@ def run_tests(argv=UNITTEST_ARGS):
         if RERUN_DISABLED_TESTS:
             other_args.append("--rerun-disabled-tests")
         if TEST_SAVE_XML:
-            other_args += ['--save-xml', args.save_xml]
+            other_args += ['--save-xml', TEST_SAVE_XML]
```

Comment:
Is TEST_XML always already set by args or could we have a fallback?

Reply:
Yes: `TEST_SAVE_XML = args.save_xml` in [line 975](https://github.com/pytorch/pytorch/pull/152170/files#diff-3f788198eb89507c88f44b077881d577e0cdecc0bdd4d7cc3c94b4b4b2dba483R975)